README.md

```diff
@@ -0,0 +1,89 @@
+# recodeflow <img src="man/figures/logo.svg" align="right" width="180"/>
+
+<!-- badges: start -->
+
+[![Lifecycle:
+development](https://img.shields.io/badge/lifecycle-maturing-blue.svg)](https://www.tidyverse.org/lifecycle/#maturing)
+[![](https://img.shields.io/cran/v/cchsflow?color=green)](https://CRAN.R-project.org/package=TBA)
+![](https://img.shields.io/github/v/release/big-life-lab/recodeflow?color=green&label=GitHub)
+[![License:
+MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
+<!-- badges: end -->
+
+# Introduction
+
+## What is `recodeflow`?
+
+`recodeflow` transforms variables from multiple data sets into harmonize variables.
+
+`recodeflow` has the basic functions and templates required to define, transform, and harmonize variables for any dataset.
+
+
+## Why should I use `recodeflow`? 
+
+Let's walk through an example. 
+
+Each year, Canadians complete a survey, and the data is collected. The data is coded.  You, a scientist, want to use data from the 10 surveys to answer two research questions **but there's a catch:**
+
+At the survey level: 
+
+- Not every survey question was asked each year.
+- The survey answers changed from year to year. 
+
+At the data level:
+
+- Questions are not coded the same from year to year (e.g., variable names differ)
+- Answers are not coded the same from year to year (e.g., height is coded in meters in some years and feet in other years)
+
+The data is messy! You start the timely and detailed process of cleaning and transforming the data. You think "there must be a better way."
+
+That's where `recodeflow` comes to the rescue. 
+
+With `recodeflow` you can clean and transform your entire dataset with a few lines of code. Even better, once you have defined your variables you have variable sheets you can reuse or share with colleagues - saving you even more time down the line.
+
+
+## How does `recodeflow` work?
+
+Use templates `variable_sheet` and `variable_details` to list your variables, and state how to transform the each variable.
```

Comment:
I can't remember the conversation but `variable_sheet` name seems a bit off because `sheet` isn't a common term in R or other languages. I thought we were going to use the term `variables` and `variables.csv`?  `table` could work. Regardless, we should be consistent and use the same term for the actual table/sheet. `variable_table` == `variable_table.csv`

Reply:
I've changed all references throughout the documentation to variables (from variables_sheet)